scripts/assemble-weights.silva_132.sh

```diff
@@ -0,0 +1,107 @@
+# This workflow script contains all commands required to generate the
+#     taxonomic weights found in readytowear/data/silva/*/
+# Note that temporary files are removed at the end of this workflow.
+#     Remove the last line of this script to save temporary files.
+
+# move your current working directory to readytowear/ or alter this line
+pdir=.
+njobs=4
+
+cdir=$pdir/data/silva_132/
+mkdir $cdir
+
+tmp=$cdir/tmp/
+mkdir $tmp
+
+qiime rescript get-silva-data \
+    --p-version '132' \
+    --p-target 'SSURef_NR99' \
+    --p-include-species-labels \
+    --o-silva-sequences $tmp/silva-132-ssu-nr99-seqs.qza \
+    --o-silva-taxonomy $tmp/silva-132-ssu-nr99-tax.qza
+
+qiime rescript cull-seqs \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs.qza \
+    --o-clean-sequences $tmp/silva-132-ssu-nr99-seqs-cleaned.qza
+
+qiime rescript filter-seqs-length-by-taxon \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs-cleaned.qza \
+    --i-taxonomy $tmp/silva-132-ssu-nr99-tax.qza \
+    --p-labels Archaea Bacteria Eukaryota \
+    --p-min-lens 900 1200 1400 \
+    --o-filtered-seqs $tmp/silva-132-ssu-nr99-seqs-filt.qza \
+    --o-discarded-seqs $tmp/silva-132-ssu-nr99-seqs-discard.qza
+
+qiime rescript dereplicate \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs-filt.qza  \
+    --i-taxa $tmp/silva-132-ssu-nr99-tax.qza \
+    --p-rank-handles 'silva' \
+    --p-mode 'uniq' \
+    --o-dereplicated-sequences $cdir/full_length/ref-seqs.qza \
+    --o-dereplicated-taxa $cdir/full_length/ref-tax.qza
```

Comment:
I don't think this is a good approach. We don't provide the outputed `ref-seqs.qza` in this repo, so the user has to know or guess that it doesn't matter if they download the ref seqs included in the standard `Silva_132_release.zip` and marry them with this version of `ref-tax.qza`. It is unlikely that they would know and not good to make them guess.

It's also unnecessary because all we have to do is uncomment the commented lines below and rerun the script.